plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveRoles.java

```diff
@@ -13,487 +13,14 @@
  */
 package io.trino.plugin.hive;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import io.trino.Session;
-import io.trino.spi.security.Identity;
-import io.trino.spi.security.SelectedRole;
-import io.trino.spi.type.Type;
-import io.trino.testing.AbstractTestQueryFramework;
-import io.trino.testing.MaterializedResult;
-import io.trino.testing.QueryRunner;
-import org.testng.annotations.AfterMethod;
 import org.testng.annotations.Test;
 
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.stream.Collectors;
-
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkState;
-import static io.trino.spi.type.VarcharType.createUnboundedVarcharType;
-import static io.trino.testing.QueryAssertions.assertContains;
-import static io.trino.testing.QueryAssertions.assertEqualsIgnoreOrder;
-import static org.testng.Assert.assertEquals;
-
 @Test(singleThreaded = true)
 public class TestHiveRoles
-        extends AbstractTestQueryFramework
+        extends AbstractTestHiveRoles
```

Comment:
Did you modify any of tests? Is it a simple extraction of abstract class? It is good to have automic changes extracted as separate commit.

Reply:
I basically changed every line.  I moved the migration to a separate commit, so the commit is still pretty large.